config/initializers/assets.rb

```diff
@@ -9,3 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( email.css )
```

Comment:
We need this in order to precompile an email.css file separate from application.css https://www.dropbox.com/s/tow13g2s01tfhkz/email-assets-ss.jpg?dl=0